.tools/ailly/docs/.aillyrc

```diff
@@ -0,0 +1,51 @@
+---
+combined: true
+overwrite: false
+---
+
+TCX SDK Code Examples
+The TCX SDK Code Examples team produces code examples that demonstrate how to automate AWS services to accomplish key user stories for developers and programmers. These code examples are quick and easy to find and use, are continually tested, and demonstrate AWS and community coding best practices.
+
+Mission
+
+We provide code examples for builders integrating AWS services into their applications and business workflows using the AWS Software Development Kits (SDKs). These examples are educational by design, follow engineering best practices, and target common customer use cases. Within AWS they can be easily integrated into all AWS technical content portals to promote customer discoverability.
+
+Vision
+
+We envision a best-in-class library of code examples for every AWS service and in every actively maintained SDK language. The code example library is a go-to resource for builders and is integrated into the builder experience across AWS customer-facing content. Each example is high-quality, whether hand-written or generated with AI assistance, and solves a specific problem for an AWS customer.
+
+Tenets
+
+These are our tenets, unless you know better ones:
```

Comment:
Think it would be helpful to add a bit about what tenets are? Especially that their hierarchical.

Reply:
/shrug - I don't really want to litigate that in _this_ PR, because I think it's "good enough" for today. As we know, we _don't_ know where the line is between "enough" and "too much" in the system prompt. I'd like to use this system prompt for the workshop, then we can evolve it?

Reply:
Sounds good.